push.sh

```diff
@@ -26,13 +26,17 @@ else
 fi
 
 # Run backend tests if the server directory has changes on the current branch and is different from origin/develop
+if ! docker info > /dev/null 2>&1; then
```

Comment:
the script will check the user is running docker to run these tests since we use a containerized db

[script source](https://stackoverflow.com/a/55283209/20660852)